morgan/__init__.py

```diff
@@ -538,6 +540,15 @@ def my_url(arg):
         nargs="?",
         help="Config file (default: <INDEX_PATH>/morgan.ini)",
     )
+    parser.add_argument(
+        "--package-type-regex",
+        dest="package_type_regex",
+        default="(whl|zip|tar.gz)",
+        type=str,
+        nargs="?",
+        const="(whl|zip|tar.gz)",
```

Comment:
```suggestion
```

I don't see a benefit of having `--package-type-regex` specified without a string afterwards. The default will use the full string anyway.